Patient Registry System/PatientRegistrySystem.API/Controllers/AccountController.cs

```diff
@@ -113,7 +113,17 @@ public async Task<IActionResult> Register([FromBody] RegistrationViewModel model
             }
             else
             {
-                return BadRequest("Input problem");
+                string messages = "";
+                foreach (var modelState in ViewData.ModelState.Values)
+                {
+                    foreach (var error in modelState.Errors)
```

Comment:
i just noticed that you have all this logic in the AccountController, we should move it out to an AccountService.